src/content/1.7/development/components/grid/tutorials/work-with-row-actions.md

```diff
@@ -0,0 +1,196 @@
+---
+title: How to use Row Actions
+menuTitle: Row Actions
+weight: 2
+---
+
+# How to use Row actions
+
+In addition to displaying data tables, Grid component also comes with a lot of additional features. One is the Row actions.
+
+## What is a Row action?
+
+A Row action is an action (e.g. Edit, View & Delete) that is applicable to a specific row in the Grid.
```

Comment:
You mean when _clicking_ on a specific row?

Reply:
No it's the list of actions available in the last column, the click action is only the one with the `clickable_row` option (and only on LinkAction for now)